src/controlSurface.cpp

```diff
@@ -82,13 +82,18 @@ class Surface: public IReaperControlSurface {
 	}
 
 	void Run() final {
-		if (settings::reportMarkersWhilePlaying && GetPlayState() & 1) {
+		if (GetPlayState() & 1) {
 			double playPos = GetPlayPosition();
 			if (playPos == this->lastPlayPos) {
 				return;
 			}
 			this->lastPlayPos = playPos;
-			this->reportMarker(playPos);
+			if (settings::reportMarkersWhilePlaying) {
+				this->reportMarker(playPos);
+			}
+			if (settings::reportTimeMovementWhilePlaying) {
```

Comment:
I'm not sure about tying this to this particular setting. This setting is currently about on-demand speech; i.e. it determines whether certain explicit actions report when playing. In contrast, this check is about whether we report something automatically even without an explicit action. In that sense, this is more like the report markers during playback option. Those two things are quite different.

Happy to be convinced otherwise, but I do think it needs convincing. I'd personally be more inclined to tie this to the markers option and rename that option to cover time selection.

Reply:
Ah ok, hadn't considered the difference between on demand and automatic reporting. I could create a new toggle just for time selection? Feels like users would be opting in for what could be quite a lot of additional speech if we tie it in with markers/regions, also given how often markers are used for navigation, I think there's a high likelihood of a marker/region and one edge of a time selection to be in the same position.

Reply:
I'm not dead against it, but I would like to understand how users will perceive these two things as inherently related. My concern is that some folks might want to know where they are if they move during playback, but not have their speech nattering at them if they don't particularly care about their selection right now. If you really think this isn't going to bother users - you'd probably know more than I would given you deal with users more often :) - perhaps we could rename this option to "Report time movement/selection during playback/recording". Otherwise, I guess we could add a new setting, though it's unfortunate that we'd have an entirely new setting covering a very narrow, single use case.

Reply:
Using it in situ today, I'm not convinced about it being tied to either of our existing options. Reckon I should make a new one called "Report time selection/loop points during playback/recording". We only report time selection if those are linked. Time selection is fundamental functionality, so to me it doesn't seem problematic that it has its own option. Any objections?

Reply:
No real objections, aside from a general reluctance to add even more settings, but that's not a useful objection. :)

I'm not following the bit about loop points though. If I'm reading the code correctly, this only looks at time selection. Loop points don't come into it at all.

Reply:
Yeah I haven't written the loop points stuff yet. It's an easy addition though. Figured they're so closely related, the new option may as well cover those too.